client/state/ui/selectors/is-blaze-enabled.js

```diff
@@ -1,7 +1,11 @@
 import { getSiteOption } from 'calypso/state/sites/selectors';
-import getSelectedSite from './get-selected-site';
-import 'calypso/state/ui/init';
 
-export default function isBlazeEnabled( state ) {
-	return !! getSiteOption( state, getSelectedSite( state ).ID, 'can_blaze' ) || false;
+/**
+ * Returns if the currently selected site can Blaze posts.
```

Comment:
We're no longer working with the selected site, we're passing a site ID instead.

```suggestion
 * Returns if the specified site can Blaze posts.
```